README.md

```diff
@@ -78,71 +66,51 @@ task count_bam {
 }
 ```
 
-From the command line, we can compile the workflow to the DNAnexus platform using the dxWDL jar file.
+From the command line, we can compile the workflow to the DNAnexus platform using the dxCompiler jar file.
+
 ```
-$ java -jar dxWDL-0.59.jar compile bam_chrom_counter.wdl -project project-xxxx
+$ java -jar dxCompiler.jar compile bam_chrom_counter.wdl -project project-xxxx
 ```
 
 This compiles the source WDL file to several platform objects.
 - A workflow `bam_chrom_counter`
 - Two applets that can be called independently: `slice_bam`, and `count_bam`
 - A few auxiliary applets that process workflow inputs, outputs, and launch the scatter.
 
-These objects are all created in the current `dx` project and folder. The generated workflow can
-be run using `dx run`. For example:
+These objects are all created in the current `dx` project and folder. The generated workflow can be run using `dx run`. For example:
+
 ```
 dx run bam_chrom_counter -i0.file=file-xxxx
 ```
 
-At runtime this looks like this:
+The compiled workflow can be executed via the DNAnexus command line client or web UI:
+
 ![this](doc/bam_chrom_counter.png)
 
 ## Strict syntax
 
-One of the compiler phases takes a workflow apart, and extracts standalone tasks and sub-workflows. This requires a lexical analysis on the WDL program. It currently uses a simple regular expression to detect task/workflow start and end. This means that a task has to adhere to the following rules:
-1. no extra text is allows after the final closing bracket
-2. within the task body, closing brackets may not start at the beginning of a line.
+dxCompiler uses [wdlTools](https://github.com/dnanexus-rnd/wdlTools), a parser that adheres strictly to the specifications. Most of the problematic automatic type conversions that are allowed by some other WDL runtime engines are not allowed by dxCompiler. Please use the command line tools in wdlTools (e.g. `check` and `lint`) to validate your WDL files before trying to compile them with dxCompiler.
```

Comment:
A note that this limits the usefulness of the `workflow_importer` app that imports workflows from Dockstore.